read-fonts/src/tables/variations.rs

```diff
@@ -577,6 +577,66 @@ impl<'a> ItemVariationStore<'a> {
         }
         Ok(((accum + 0x8000) >> 16) as i32)
     }
+
+    /// Computes the delta value in floating point for the specified index and set
+    /// of normalized variation coordinates.
+    ///
+    /// This requires a type parameter specifying the type of target value for the
+    /// delta.
+    pub fn compute_delta_f32<T: ItemDeltaTarget>(
```

Comment:
Is it possible to add some standalone file level tests for this?

Reply:
Will do!

Reply:
We don't have a reference for correctness here so I added a sanity test to ensure that the floating point results are within a reasonable margin to the delta produced by the FT compatible computation. Specifically, it ensures that either the rounded or truncated floating point delta matches the FT compatible delta. We have to account for both because fractional accumulation can push us to either side of the 0.5 boundary in some cases.